### ISSUE TITLE
DropDownList binding to child objects

### ISSUE THREAD

**Issue by lucasheight** [telerik/kendo-angular#3038]
**Describe the bug**
Dropdownlist @progress/kendo-angular-dropdowns 4.3.0 does not bind correctly to child objects on the array:
```typescript
import { Component } from '@angular/core';

@Component({
  selector: 'my-app',
  template: `
    <div class="example-wrapper">
        <kendo-dropdownlist [data]="listItems"
        textField="details.name"
        valueField="id"
        ></kendo-dropdownlist>
    </div>
  `
})
export class AppComponent {
    public listItems: Array<{id:number, name:string,details:{name:string}}> = [
      {id:1, name:"Item 1",details:{name:"detailed name 1"}},
      {id:2,name: "Item 2", details:{name:"detailed name 2"}},
      {id:3,name: "Item 3", details:{name:"detailed name 3"}}];
}

```
Example: https://stackblitz.com/edit/angular-ncpssr?file=app/app.component.ts
**To Reproduce**
In the example above the list is populated, but the text is not visible

**Comment by dimitar-pechev** [comment]
The DropDowns currently do not support reading the text and value from nested properties. The quick workaround for now is to flatten your data and pull the targeted properties to the top level of the objects.

Related to https://github.com/telerik/kendo-angular/issues/2571.

**Comment by lucasheight** [comment]
@dimitar-pechev Thanks, thats what I ended up doing. I was surprised that it didn't as the grid does.

**Comment by ipeshev** [comment]
Feature shipped with @progress/kendo-angular-dropdowns@5.3.0.
